source/localizable/getting-started/quick-start.md

```diff
@@ -211,13 +211,6 @@ curly braces (`{{component}}`). We're going to tell our component:
    provide this route's `model` as the list of people.
 
 ```app/templates/scientists.hbs{-1,-2,-3,-4,-5,-6,-7,+8}
```

Comment:
Also, all this started happening since https://github.com/emberjs/guides/commit/d2b2c49f4828fcfec3bfcf0d025755d53a27976e, before the background was white so those green/red applied correctly. See for instance how was the style before the mentioned commit and how looks in the blog http://emberjs.com/blog/2016/01/12/ember-data-2-3-released.html